mcumgr-core/src/main/java/io/runtime/mcumgr/util/CBOR.java

```diff
@@ -36,6 +36,11 @@ public static String toString(byte[] data) throws IOException {
         return mapper.readTree(data).toString();
     }
 
+    public static String toString(byte[] data, int offset) throws IOException {
```

Comment:
Is this being used?

Reply:
Yes, to parse CBOR from a payload that contains a header (8 bytes). Otherwise, I'd have to copy the array from an offset to pass to a method that actually accepts an offset. This just looks weird, as it's a single method with this offset in this class, but I think it's less ugly and faster.

Reply:
In fact... I no longer use it. I must have refactored the code and forgot about this. Removing!

Reply:
After all I restored this method. See my last comment.